node/test/util/read_repo_ast_util.js

```diff
@@ -383,12 +383,24 @@ describe("readRAST", function () {
         const expected = new RepoAST({
             commits: commits,
             branches: { "master": commit },
-            head: null,
+            head: commit,
             currentBranchName: "master",
+            bare: true,
         });
         RepoASTUtil.assertEqualASTs(ast, expected);
     }));
 
+    // For now, we care about an explicit headless state, we're not concerned
```

Comment:
I don't know what's up with Github, but if you look here: https://github.com/twosigma/git-meta/pull/165/files, you'll see that the quotes are already there.